File: airbyte-commons-temporal-core/src/main/java/io/airbyte/commons/temporal/exception/SizeLimitException.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.temporal.exception;

/**
 * Exception when an activity fails because the output size exceeds Temporal limits.
 */
public class SizeLimitException extends RuntimeException {

  public SizeLimitException(final String message) {
    super(message);
  }

}

File: airbyte-commons-temporal-core/src/main/kotlin/io/airbyte/commons/temporal/exception/SizeLimitException.kt
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.temporal.exception

/**
 * Exception when an activity fails because the output size exceeds Temporal limits.
 */
class SizeLimitException(
  message: String?,
) : RuntimeException(message)
